nifi-api/src/main/java/org/apache/nifi/counter/Counter.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.controller;
+package org.apache.nifi.counter;
```

Comment:
Can/should we make these kinds of API changes at a minor/dot release? I think it might have to wait for a major release?